paper_slides/code/tables.lyx

```diff
@@ -94,7 +94,7 @@ status open
 \begin_inset Caption Standard
 
 \begin_layout Plain Layout
-Table Title
+Regression results for post-TV
```

Comment:
Title looks good; reminder we use Title Case for all titles. So, instead of "Regression results for post-TV" we would have "Regression Results for post-TV".